docs/functionalityValidation.md

```diff
@@ -0,0 +1,16 @@
+# Functionality Validation
+
+## Completed task
+
+A functionnality (or development task) is said to be finished if it meets all these conditions:
+* At least one code review has been done.
+* Integration tests have been performed.
+* The new development has been integrated into git.
+* The functionality has been tested and validated by the Product Owner.
+* The developper documentation has been updated (well-commented code, test examples, UML, etc...)
+
```

Comment:
Maybe add that during it could also is the production of a video, a demo, slides for the revue of sprint.
It also be planned at the beginning of the sprint.